src/Vulkan/RHI/VkBuffer.cpp

```diff
@@ -42,7 +42,7 @@ static ::vk::BufferUsageFlags GetVkBufferUsageFromDesc(const BufferDescription&
 
     if (!flags)
     {
-        VEX_LOG(Fatal, "Buffer usage not supported by VulkanRHI.");
+        flags |= eTransferSrc;
```

Comment:
Shouldn't this always be true? In case for example we want to duplicate a buffer?